blocks/subquery/edit.tsx

```diff
@@ -0,0 +1,273 @@
+/* eslint-disable camelcase */
+import { useDebounce } from '@uidotdev/usehooks';
+import apiFetch from '@wordpress/api-fetch';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+import {
+  useEffect,
+} from '@wordpress/element';
```

Comment:
WordPress has moved away from importing from `@wordpress/element` to importing directly from React, and we should do the same